### PR TITLE
MTGJson downloader

### PR DESCRIPTION
**Pull request by ahmetabdi**

### CODE REVIEW

File: lib/mtgdb/downloaders/downloader.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      FileUtils.mkdir_p(directory)
+    end
+
+    def delete_files(directory)
+      Dir.glob(directory).each do |fname|
+        File.delete(fname)
+      end
+    end
+
+    def delete_file(location)
```
Comment (steview) [inline lib/mtgdb/downloaders/downloader.rb:13]:
```
def delete_file(path)
  File.delete(path) if File.exists?(path)
end

File: lib/mtgdb/downloaders/mtg_json_downloader.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class MTGDB::Downloaders::MtgJsonDownloader < MTGDB::Downloaders::Downloader
+  def self.run
+    create_directory('tmp/importer')
+    delete_files('tmp/importer/*.json')
+    delete_files('tmp/importer/*.zip')
+
+    uri = URI("https://mtgjson.com/json/AllSets-x.json.zip")
+    zipped_folder = Net::HTTP.get(uri)
+
+    File.open("tmp/importer/AllSets-x.json.zip", 'wb') do |file|
```
Comment (steview) [inline lib/mtgdb/downloaders/mtg_json_downloader.rb:12]: " 😢

File: lib/mtgdb/downloaders/mtg_json_downloader.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      file.write(zipped_folder)
+    end
+
+    zip_file = Zip::File.open("tmp/importer/AllSets-x.json.zip")
```
Comment (steview) [inline lib/mtgdb/downloaders/mtg_json_downloader.rb:16]: " 😢

File: lib/mtgdb/downloaders/mtg_json_downloader.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    zip_file.each do |file|
+      file.extract("tmp/importer/AllSets-x.json")
```
Comment (steview) [inline lib/mtgdb/downloaders/mtg_json_downloader.rb:18]: " 😢

File: lib/mtgdb/downloaders/mtg_json_downloader.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    end
+
+    delete_file('tmp/importer/AllSets-x.json.zip')
+
+    file = File.read("tmp/importer/AllSets-x.json")
```
Comment (steview) [inline lib/mtgdb/downloaders/mtg_json_downloader.rb:23]: " 😢

**Comment by ahmetabdi** [comment]
😄 I thought you liked double quotes?

**Pull request closed without merging**